### ISSUE TITLE
[Request] Filter for Potential Awakenings

### ISSUE THREAD

**Issue by dumdummuoi** [madcowfred/padherder-issues#531]
Currently the awakenings filter only shows monsters that have that awakening already activated. It would be nice to have a toggle to also show potential awakenings so I know which monsters can potentially have the awakening(s) I need if I invest EXP/evo mats/Tamadra into it.

**Comment by Marco-PF** [comment]
I agree. Just like there is a "Need XP" field, a "Need Awakening(s)" would be useful."
Good idea!

**Comment by madcowfred** [comment]
This seems like something useful to add, I'll have to see what the performance is like though.
